src/main/java/school/lemon/changerequest/java/introduction/hw1/Task4.java

```diff
@@ -4,6 +4,23 @@
 public class Task4 {
 
     public static double getSumOfGreatest(double a, double b, double c) {
-        return 0;
+
+        double firstMax;
```

Comment:
It's too complicated.
You can go from the back side, find the smallest number in one `if` statement and return summ of the others.